stories/feed-list/feed-list.stories.js

```diff
@@ -0,0 +1,562 @@
+    }
+};
+
+export const borderless = () => `<ul class="fd-feed-list fd-feed-list--group fd-feed-list--no-border" aria-label="Feed List Item group example">
+    <li class="fd-feed-list__body">
+        <div
+            class="fd-avatar fd-avatar--s fd-avatar--placeholder fd-feed-list__thumb" 
+            role="img" 
+            aria-label="John Doe"
+            title="John Doe">
+                <i class="sap-icon--person-placeholder"></i>
+        </div> 
+        <div class="fd-feed-list__content">
+            <p class="fd-feed-list__text">
+                <span class="fd-feed-list__name">Cruz: </span>
+                <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus.        
+            </p>
+            <div class="fd-feed-list__footer">
+                <small class="fd-feed-list__footer--byline">Aug 21, 2020</small>
+                <small class="fd-feed-list__footer--byline">Reply</small>
+            </div>
+        </div>
+   </li>
+   
+    <li class="fd-feed-list__body">
+        <div 
+            class="fd-avatar fd-avatar--s fd-avatar--thumbnail fd-feed-list__thumb" 
+            style="background-image: url(https://avatars.dicebear.com/api/human/fe.svg?m=6&b=%23f5f0f0&mood[]=surprised)" 
+            role="img" 
+            aria-label="John Doe"
+            title="John Doe">    
+        </div>
+     
+        <div class="fd-feed-list__content">
+            <p class="fd-feed-list__text">
+                <span class="fd-feed-list__name">Joe Doe: </span>
+                <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus.        
+            </p>
+            <div class="fd-feed-list__footer">
+                <small class="fd-feed-list__footer--byline">Aug 22, 2020</small>
+                <small class="fd-feed-list__footer--byline">Reply</small>
+            </div>
+        </div>     
+    </li>
+    
+    <li class="fd-feed-list__body">
+        <div
+            class="fd-avatar fd-avatar--s fd-feed-list__thumb"
+            style="background-image: url(https://avatars.dicebear.com/api/male/john.svg?mood[]=happy)" 
+            role="img" 
+            aria-label="John Doe"
+            title="John Doe"
+        ></div> 
+        <div class="fd-feed-list__content">
+            <p class="fd-feed-list__text">
+                <span class="fd-feed-list__name">Sanchez: </span>
+                <span>Maecenas convallis velit quis felis dictum, in ultrices quam faucibus.        
+            </p>
+            <div class="fd-feed-list__footer">
+                <small class="fd-feed-list__footer--byline">Aug 20, 2020</small>
+                <small class="fd-feed-list__footer--byline">Reply</small>
+            </div>
+        </div>
+   </li>
+</ul>
+`;
+
+borderless.storyName = 'Borderless group';
+borderless.parameters = {
+    docs: {
+        storyDescription: 'The example below displays a feed list item without a separated border'

```

Comment:
When feed list items are displayed in a group, they are usually separated by borders. However, to remove the borders, add the `fd-feed-list--no-border` modifier class to the main element.